### PR TITLE
Footer

### PR DESCRIPTION
**Pull request by nancyfontanez**
## Summary
Styled footer as mockup.
## Checklist
- [ ] fixed|updated|added unit tests and integration tests for each feature (if applicable).
- [x] tested locally.
- [x] No error nor warning in the console.

[upload the screenshot here] (optional)
15 10 2022_19 10 01_REC

### CODE REVIEW

**Review by cherylli — CHANGES_REQUESTED** [review]
Looks good but a couple of things I noticed are different from the design
- the footer in the design has no gradient
- the footer width is 100% width in the design

**Comment by nancyfontanez** [comment]
> Looks good but a couple of things I noticed are different from the design
> 
> * the footer in the design has no gradient
Figma's design shows a gradient in the footer. Am I looking at the right one?
16 10 2022_15 25 19_REC

> * the footer width is 100% width in the design
I will work on this ASAP.

**Comment by cherylli** [comment]
yes sorry you're right, there's a gradient. one of my monitors is weird, and I only inspected by eyes

**Comment by nancyfontanez** [comment]
#80 Footer width fixed.
15 10 2022_19 10 01_REC

**Comment by nancyfontanez** [comment]
> yes sorry you're right, there's a gradient. one of my monitors is weird, and I only inspected by eyes

#80 Footer width fixed.
17 10 2022_19 41 04_REC

**Review by cherylli — APPROVED** [review]
Looks good!

File: client/src/Components/Footer/Footer.module.css [hunk 1]
```diff
@@ -1,33 +1,42 @@
 .footer {
-  background-color: #E6E6E6;
+  width: 100vw;
+  height: 219px;
+  background: rgb(10,14,36);
+  background: linear-gradient(90deg, rgba(10,14,36,1) 0%, rgba(0,64,87,1) 90%);
```
Comment (klezi10) [inline client/src/Components/Footer/Footer.module.css:5]: Is there a need for the first background property? It gets overridden by the second one anyways?

**Review by klezi10 — APPROVED** [review]

**Pull request merged**